src/components/app/landing/login/LoginModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { Form, Input, Button, Modal, notification } from 'antd';
+import classes from './LoginModal.module.css';
+import Credential from '../../../../types/user/credentials';
+import { useState, useEffect } from 'react';
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const LoginModal = (props: Props) => {
+  const [formIsValid, setFormIsValid] = useState<boolean>(false);
+  const [enteredEmail, setEnteredEmail] = useState<string>('');
+  const [enteredPassword, setEnteredPassword] = useState<string>('');
+
+  useEffect(() => {
+    setFormIsValid(
+      enteredEmail.includes('@') && enteredPassword.trim().length > 6,
+    );
+  }, [enteredEmail, enteredPassword]);
```

Comment:
@aritrasen12345 use antd form validation and control instead of manual. i.e. use `rules` props on                 `<Form.Item></Form.Item>`